apps/api-documenter/src/yaml/OfficeYamlDocumenter.ts

```diff
@@ -148,4 +147,16 @@ export class OfficeYamlDocumenter extends YamlDocumenter {
     }
     return text;
   }
+
+  private _generateSnippetText(snippets: string[]): string {
+    let text: string = '\n\n#### Examples\n';
+    for (const snippet of snippets) {
+      if (snippet.search(/await/) === -1) {
```

Comment:
>await [](start = 26, length = 5)

This is kind of an arbitrary distinction between TS and JS. TS supports the `await` keyword.

Reply:
Agreed. This pre-dates me, so I'll open a separate work item to put some more accurate logic in there at a future time.